_data/education.yml

```diff
@@ -5,6 +5,12 @@ websites:
       img: academicearth.png
       tfa: No
 
+    - name: Antagning (University Admissions in Sweden)
```

Comment:
They seem to want to call themselves `Antagning.se`. Please change the name to that.